### PR TITLE
ev3dev: allow fluent style API use

### LINKED ISSUE #2: ev3dev: consider allowing fluent style
If each device type holds an `err error` field and has an `Err() error` method, then setters can return the device pointer. Attribute reader and writer methods would return early if the `err` field if non-nil, so a call to `Err` should reset the `err` field to nil.

### PR DESCRIPTION
**Pull request by kortschak**
Fixes #2.

### CODE REVIEW

**Comment by kortschak** [comment]
@vladimir-ch Please take a look at the package comment explaining call chains (ignore the code - I'm reasonably confident in it and until I add testing I'm not that fussed).

File: ev3dev/ev3dev.go [hunk 1]
```diff
@@ -4,6 +4,20 @@
 
 // Package ev3dev provides low level access to the ev3dev control and sensor drivers.
 // See documentation at http://www.ev3dev.org/docs/drivers/.
+//
+// The API provide in the ev3dev package allows fluent chaining of action calls.
```
Comment (vladimir-ch) [inline ev3dev/ev3dev.go]: s/provide/provided/

File: ev3dev/ev3dev.go [hunk 2]
```diff
@@ -4,6 +4,20 @@
...
+// Methods for each of the device handle types are split into to classes: action
```
Comment (vladimir-ch) [inline ev3dev/ev3dev.go]: s/to/two/

**Comment by vladimir-ch** [comment]
Without looking at the code I can imagine from the description how the call chains work, so I think that the explanation is good.

File: ev3dev/ev3dev.go (hunk 2 above)
Comment (kortschak) [inline ev3dev/ev3dev.go]: Done

**Comment by kortschak** [comment]
Thanks.

**Pull request merged**